app/pulp/app/serializers/user.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import serializers
+
+from pulp.app import models
+from pulp.app.serializers import ModelSerializer
+
+
+class UserSerializer(ModelSerializer):
+    username = serializers.CharField(
+        help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
+    )
+
+    is_superuser = serializers.BooleanField(
+        help_text="Designates that this user has all permissions without explicitly assigning "
+                  "them."
+    )
+
+    groups = serializers.ManyRelatedField(
+        help_text="The groups this user belongs to. A user will get all permissions granted to "
+                  "each of their groups."
+    )
```

Comment:
We don't have any viewsets for groups and permissions. So I am not sure that I can add HyperlinkedRelatedFields for either of them. I am starting to think that I should remove these from the serializer from now.